hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -486,7 +486,7 @@ spec = parallel $
                     Blockfrost
                       BlockfrostOptions
                         { projectPath = "baz"
-                        , queryTimeout = 10
+                        , queryTimeout = 20
```

Comment:
i think we should have a defaultBlockfrostQueryTimeout defined somewhere and be used here